config.plist/alder-lake.md

```diff
@@ -0,0 +1,793 @@
+| **[SSDT-SBUS-MCHC](https://dortania.github.io/Getting-Started-With-ACPI/)** | May not be needed for everyone. Fixes AppleSMBus support, see [Getting Started with ACPI](https://dortania.github.io/Getting-Started-With-ACPI/) for more details. |
+| **[SSDT-BRG0](https://dortania.github.io/Getting-Started-With-ACPI/)** | Needed on some systems to restore missing devices to your ACPI tables, which fixes DeviceProperties injection issues. See [Getting Started with ACPI](https://dortania.github.io/Getting-Started-With-ACPI) for more details.
+| **[SSDT-USBW](https://dortania.github.io/Getting-Started-With-ACPI/)** | Companion to the USBWakeFixup kext, see [Getting Started with ACPI](https://dortania.github.io/Getting-Started-With-ACPI/) for more details. |
+
+Note that you **should not** add your generated `DSDT.aml` here, it is already in your firmware. So if present, remove the entry for it in your `config.plist` and under EFI/OC/ACPI.
+
+For those wanting a deeper dive into dumping your DSDT, how to make these SSDTs, and compiling them, please see the [**Getting started with ACPI**](https://dortania.github.io/Getting-Started-With-ACPI/) **page.** Compiled SSDTs have a **.aml** extension(Assembled) and will go into the `EFI/OC/ACPI` folder and **must** be specified in your config under `ACPI -> Add` as well.
+
+:::
+
+### Delete
+
+This blocks certain ACPI tables from loading, for us we can ignore this.
+
+### Patch
+
+This section allows us to dynamically modify parts of the ACPI (DSDT, SSDT, etc.) via OpenCore. For us, our patches are handled by our SSDTs. This is a much cleaner solution as this will allow us to boot Windows and other OSes with OpenCore
+
+### Quirks
+
+Settings relating to ACPI, leave everything here as default as we have no use for these quirks.
+
+## Booter
+
+![Booter](../images/config/config-universal/hedt-booter.png)
+
+This section is dedicated to quirks relating to boot.efi patching with OpenRuntime, the replacement for AptioMemoryFix.efi
+
+### MmioWhitelist
+
+This section is allowing devices to be passthrough to macOS that are generally ignored, for us we can ignore this section.
+
+### Quirks
+
+::: tip Info
+Settings relating to boot.efi patching and firmware fixes, for us, we need to change the following:
+
+| Quirk | Enabled |
+| :--- | :--- |
+| DevirtualiseMmio | YES |
+| EnableWriteUnprotector | NO |
+| ProtectUefiServices | YES |
+| RebuildAppleMemoryMap | YES |
+| ResizeAppleGpuBars | -1 | If your firmware supports increasing GPU BAR sizes (ie Resizable BAR support), set this to `0` |
+| SetupVirtualMap | NO |
+| SyncRuntimePermissions | YES |
+:::
+
+::: details More in-depth Info
+
+* **AvoidRuntimeDefrag**: YES
+  * Fixes UEFI runtime services like date, time, NVRAM, power control, etc.
+* **DevirtualiseMmio**: YES
+  * Reduces Stolen Memory Footprint, expands options for `slide=N` values and very helpful with fixing Memory Allocation issues , requires `ProtectUefiServices` as well for Z490.
```

Comment:
Not sure what current gen is, don't think it's Z4xx though

Reply:
Must have missed this when I copied the Comet Lake guide. Resolved in b3c92e3.